index.html

```diff
@@ -1184,6 +1184,152 @@ <h4>Proof Verification (Ed25519Signature2020)</h4>
 
     <section class="appendix informative">
       <h2>Test Vectors</h2>
+      <section>
+        <h3>Representation: eddsa-2022</h3>
+        <p>
+The signer needs to generate a private/public key pair with the private key used
+for signing and the public key made available for verification. The
+[[MULTIBASE]]/[[MULTICODEC]] representation for the public key, <code>ed25519-pub</code>,
+and the representation for the private key, <code>ed25519-priv</code>, are shown below.
+        </p>
+        <pre class="example" title="Private and Public keys for Signature">
+{
+    publicKeyMultibase: "z6MkrJVnaZkeFzdQyMZu1cgjg7k1pZZ6pvBQ7XJPt4swbTQ2",
+    privateKeyMultibase: "z3u2en7t5LR2WtQH5PfFqMqwVHBeXouLzo6haApm8XHqvjxq"
+}
+        </pre>
+
+        <p>
+Signing begins with a credential without an attached proof, which is converted
+to canonical form, and then hashed, as shown in the following three examples.
+        </p>
+
+        <pre class="example" title="Credential without Proof" data-include="TestVectors/unsigned.json" 
+        data-include-format="text"></pre>
+
+        <pre class="example" title="Canonical Credential without Proof" data-include="TestVectors/eddsa-2022/canonDocDataInt.txt"
+        data-include-format="text"></pre>
+
+
+      <pre class="example" title="Hash of Canonical Credential without Proof (hex)"
+      data-include="TestVectors/eddsa-2022/docHashDataInt.txt" data-include-format="text"></pre>
+
+        <p>
+The next step is to take the proof options document, convert it to canonical form,
+and obtain its hash, as shown in the next three examples.
+        </p>
+
+      <pre class="example" title="Proof Options Document" 
+      data-include="TestVectors/eddsa-2022/proofConfigDataInt.json" data-include-format="text"></pre>
+
+      <pre class="example" title="Canonical Proof Options Document"
+      data-include="TestVectors/eddsa-2022/proofCanonDataInt.txt" data-include-format="text"></pre>
+
+      <pre class="example" title="Hash of Canonical Proof Options Document (hex)"
+      data-include="TestVectors/eddsa-2022/proofHashDataInt.txt" data-include-format="text"></pre>
+
+        <p>
+Finally, we combine the two hashes, use the private key with the combined hash to
```

Comment:
How are the hashes combined? Is it a simple concatenation? Should say whatever it is... 

Also, if important (as it seems it is), the order of the hashes in the combination should be specified.

(These answers may be obvious in the included text files, but [they're not visible in the PR-preview](https://pr-preview.s3.amazonaws.com/Wind4Greg/vc-di-eddsa/pull/29.html#example-combine-hashes-of-proof-options-and-credential-hex) nor do they appear to be [within this repo](https://github.com/w3c/vc-di-eddsa/search?q=combinedHashEdSig.txt).)

Reply:
How much should we repeat the algorithm steps given in the normative text? These should match the steps of section 3. Can add more text. Kind of stuck waiting on agreement on `@context` field for base document.

Cheers Greg

Reply:
A link from each shortened step description here, back to the detailed step description in the normative text (which details would presumably answer my questions here), would help people who skip ahead to the examples (as many will do, no matter the extent of harm they'd potentially be doing themselves).

As to these particular questions, it seems to me that saying "we concatenate hash x and hash y in that order" would be substantially clearer than "we combine the two hashes" without adding much text — and linking back to the normative text would still be beneficial.